### PR TITLE
Make getSupportsOpCodeForAutoSIMD() statically accessible

### PR DESCRIPTION
**Pull request by BradleyWood**
Signed-off-by: BradleyWood <bradley.wood@ibm.com>

### CODE REVIEW

**Comment by BradleyWood** [comment]
Jenkins build all

**Comment by BradleyWood** [comment]
Jenkins build all

**Comment by BradleyWood** [comment]
Jenkins build all

**Comment by BradleyWood** [comment]
FYI @0xdaryl, can you review?

File: compiler/p/codegen/OMRCodeGenerator.cpp [hunk 1]
```diff
@@ -1859,6 +1859,11 @@ bool OMR::Power::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode
    return false;
    }
 
+bool OMR::Power::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode)
+   {
+   return getSupportsOpCodeForAutoSIMD(self()->comp()->target().cpu, opcode);
```
Comment (0xdaryl) [inline compiler/p/codegen/OMRCodeGenerator.cpp:1864]: Same comment about `TR::` namespace.

File: compiler/x/codegen/OMRCodeGenerator.cpp [hunk 2]
```diff
@@ -1081,6 +1079,11 @@ OMR::X86::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode)
    return false;
    }
 
+bool
+OMR::X86::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode)
+   {
+   return getSupportsOpCodeForAutoSIMD(self()->comp()->target().cpu, opcode);
```
Comment (0xdaryl) [inline compiler/x/codegen/OMRCodeGenerator.cpp:1085]: Same comment about `TR::` namespace.

File: compiler/z/codegen/OMRCodeGenerator.cpp [hunk 3]
```diff
@@ -4788,6 +4787,11 @@ bool OMR::Z::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode)
    return false;
    }
 
+bool OMR::Z::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode)
+   {
+   return getSupportsOpCodeForAutoSIMD(self()->comp()->target().cpu, opcode, self()->getSupportsVectorRegisters());
```
Comment (0xdaryl) [inline compiler/z/codegen/OMRCodeGenerator.cpp:4792]: Same comment about `TR::` namespace.

**Comment by BradleyWood** [comment]
Jenkins build all

**Comment by BradleyWood** [comment]
Jenkins build all

**Review by 0xdaryl — APPROVED** [review]

**Pull request merged**